.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,18 @@
+{
+  "image": "ghcr.io/gregac/sonata-software-container:latest",
+  "remoteUser": "sonata",
+  "containerUser": "sonata",
+  "onCreateCommand": "cd /workspaces && git clone https://github.com/lowRISC/sonata-system.git",
+  "settings": {
+    "terminal.integrated.profiles.linux": {
+      "Nix devshell": {
+        "path": "nix",
+        "args": [
+           "develop",
+           "github:lowRISC/lowrisc-nix#cheriot"
+        ]
+      }
+    },
+    "terminal.integrated.defaultProfile.linux": "Nix devshell"
```

Comment:
I really want the terminal you get when you open the codespace to already be a nix devshell, though haven't got this working though it is at least the default type when you create a new terminal.

Reply:
I found a only slightly-hack way to do this with a vscode plugin : https://github.com/lowRISC/sonata-software/compare/main...hcallahan-lowrisc:sonata-software:pr1
I don't believe the codespaces settings schema has a built in way to do this.